src/styles/components/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+
+  padding: 50px $calendar-padding $calendar-padding;
+  width: getCalendarWidth($day-sizes, $calendar-gap, $calendar-padding, 7);
+
+  @for $date from 28 through 31 {
+    &--month-length--#{$date} {
+      & .calendar__day:nth-child(n + #{$date + 1}) {
+        display: none;
+      }
+    }
+  }
+}
+
+.calendar__day {
+  width: $day-sizes;
+  height: $day-sizes;
+  position: relative;
+
+  background-color: #eee;
+  border: 1px solid #000;
+  transition: background-color 500ms 100ms, transform 500ms 100ms;
+
+  &:hover {
+    cursor: pointer;
```

Comment:
`cursor: pointer;` shoudn't be in the hover move it to the `calendar__day` selector